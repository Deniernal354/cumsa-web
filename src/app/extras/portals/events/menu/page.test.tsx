import { render } from "@testing-library/react";
import EventsMenu from "./page";

test("EventsMenu snapshot", () => {
  const { asFragment } = render(<EventsMenu />);
  expect(asFragment()).toMatchSnapshot();
});
