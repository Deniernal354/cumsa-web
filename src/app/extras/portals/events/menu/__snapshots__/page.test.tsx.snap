// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventsMenu snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center gap-8"
  >
    <div
      class="w-full max-w-4xl self-stretch"
    >
      <a
        class="group inline-flex items-center gap-2 px-4 py-2 text-sm font-medium text-[var(--navy)] bg-white/80 backdrop-blur-sm border border-[color:var(--navy)]/20 rounded-lg shadow-sm hover:bg-[color:var(--blue)]/10 hover:text-[var(--red)] hover:border-[color:var(--blue)]/40 hover:shadow-md transition-all duration-200 ease-in-out"
        href="/api/events/logout"
      >
        <svg
          class="w-4 h-4"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M304 336v40a40 40 0 0 1-40 40H104a40 40 0 0 1-40-40V136a40 40 0 0 1 40-40h152c22.09 0 48 17.91 48 40v40m64 160 80-80-80-80m-192 80h256"
            fill="none"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="32"
          />
        </svg>
        Logout
      </a>
    </div>
    <div
      class="text-center max-w-xl animate-fade-in"
    >
      <h2
        class="text-3xl font-extrabold text-[var(--blue)] tracking-tight"
      >
        Manage Events
      </h2>
      <p
        class="mt-2 text-[var(--navy)]/80"
      >
        Create, edit, curate and keep the events board fresh.
      </p>
    </div>
    <div
      class="grid grid-cols-1 sm:grid-cols-2 gap-5 w-full max-w-2xl"
    >
      <a
        class="card p-6 transition-base hover:shadow animate-slide-in-left flex flex-col items-center text-center"
        href="/extras/portals/events/add"
      >
        <span
          class="font-semibold"
        >
          Add Event
        </span>
      </a>
      <a
        class="card p-6 transition-base hover:shadow animate-slide-in-right flex flex-col items-center text-center"
        href="/extras/portals/events/edit"
      >
        <span
          class="font-semibold"
        >
          Edit Event
        </span>
      </a>
      <a
        class="card p-6 transition-base hover:shadow animate-slide-in-left flex flex-col items-center text-center"
        href="/extras/portals/events/remove"
      >
        <span
          class="font-semibold"
        >
          Remove Event
        </span>
      </a>
      <a
        class="card p-6 transition-base hover:shadow animate-slide-in-right flex flex-col items-center text-center"
        href="/extras/portals/events/image-manager"
      >
        <span
          class="font-semibold"
        >
          Image Manager
        </span>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
